app/views/users/index.html.erb

```diff
@@ -9,6 +9,7 @@
           <th>Email</th>
           <th></th>
           <th></th>
+          <th></th>
```

Comment:
Even if I want to make it explicit that there are a couple of columns? I always thought colspan was used if content was spanning over multiple columns, right?